### PR TITLE
Foundry v10 jc

### PR DESCRIPTION
**Pull request by johnc303**
- Fixed more `.data` paths (in the process, fixing the obvious issues with data migrations).
- Code tidy up.
- Created draft data migration class (WIP)
- Updated system info.

### CODE REVIEW

File: module/migrations/PrimeMigration_0_3_1_to_0_4_0.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+export class PrimeMigration_0_3_1_to_0_4_0
```
Comment (biz812) [inline module/migrations/PrimeMigration_0_3_1_to_0_4_0.js:1]: I take it this is just a placeholder for the data munging logic we're likely going to need to add?

File: system.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "prime",
   "title": "Prime system",
   "description": "The foundry VTT system of the Prime RPG as used in the Possession world.",
-  "version": "0.3.1",
+  "version": "0.3.2",
```
Comment (biz812) [inline system.json]: We have a new migration for 0.3.1-->0.4.0, this patch version bump to 0.3.2 isn't covered by the case statement.

File: module/migrations/PrimeMigration_0_3_1_to_0_4_0.js (hunk 1 above)
Comment (johnc303) [reply module/migrations/PrimeMigration_0_3_1_to_0_4_0.js:1]: Yup - probably pint some calls to migrate v1 actors to v2 inside here. I'm working on some bits with that right now, probably do to have a chat about the final shape of things but I'm just making up some tools at the moment.

File: system.json (hunk 2 above)
Comment (johnc303) [reply system.json]: Haha, good spot, I'll sync. those up.

Comment (johnc303) [reply system.json]: Should be all good now

Comment (biz812) [reply system.json]: Cheers!

**Comment by biz812** [comment]
Ship it ⛵

**Pull request merged**